src/index.ts

```diff
@@ -17,12 +17,25 @@ async function run() {
 	const token = core.getInput("token", { required: true });
 	const octokit = github.getOctokit(token);
 	const pr = github.context.payload.issue?.number;
-	const author = github.context.payload.comment?.user.login;
```

Comment:
I think it's possible to get the PR author from [`github.context.payload.issue.user.login`](https://github.com/kevinzunigacuellar/coauthor-action/actions/runs/7851702670/job/21429140494#step:3:151)

If that doesn't work, the graphql query also includes the [PR author](https://github.com/kevinzunigacuellar/coauthor-action/blob/main/src/constants.ts#L5-L7) so it's not necessary to make a second call.